content/kubermatic/master/installation/add_seed_cluster/_index.en.md

```diff
@@ -95,30 +48,26 @@ It's also advisable to install the `s3-exporter` Helm chart, as it provides basi
 With this you can install the chart:
 
 ```bash
-cd kubermatic-installer
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace nodeport-proxy nodeport-proxy charts/nodeport-proxy/
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace minio minio charts/minio/
 helm --tiller-namespace kubermatic upgrade --install --values /path/to/your/helm-values.yaml --namespace s3-exporter s3-exporter charts/s3-exporter/
 ```
 
-### 3. Create Seed Resource
 
-To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource
-**in the master cluster**.
 
-{{% notice warning %}}
-Do not install the `kubermatic-operator` chart into seed clusters. It's possible to run master and seed in the same
-Kubernetes cluster, but this still means only a single operator is deployed into the shared cluster.
-{{% /notice %}}
+
+### Add the Seed Resource
+
+To connect the new seed cluster with the master, you need to create a kubeconfig Secret and a Seed resource.
+
+You will add the **master cluster** as **seed cluster**-
```

Comment:
Why is the sentence ending in - ?

Reply:
Also the previous sentence and the next are kind of connected logically, the `you will add the seed as...` line should probably be moved up to be the section's first statement.